Jenkinsfile

```diff
@@ -17,7 +17,7 @@ pipeline {
     BENCHMARK_SECRET  = 'secret/apm-team/ci/benchmark-cloud'
   }
   options {
-    timeout(time: 75, unit: 'MINUTES')
+    timeout(activity: true, time: 1, unit: 'HOURS')
```

Comment:
`Timeout after 1 hour of no activity in logs for pipeline instead of absolute duration.`

Reply:
if a command is showing an error every 30 min, the job could never end. I am not sure is a good option.